Source/WTF/wtf/glib/GSpanExtras.h

```diff
@@ -19,11 +19,40 @@
 
 #pragma once
 
+#include <wtf/MallocSpan.h>
 #include <wtf/StdLibExtras.h>
 #include <wtf/glib/GRefPtr.h>
+#include <wtf/glib/GUniquePtr.h>
+
+extern "C" {
+typedef char** GStrv;
+void g_strfreev(GStrv);
+}
 
 namespace WTF {
 
+template <typename T>
+struct GMalloc {
+    static void free(T* ptr) { g_free(ptr); }
+};
```

Comment:
Maybe we should just go ahead and add malloc, zeroedMalloc, tryMalloc, tryZeroedMalloc, and realloc now, even though they're not used yet. They should all be trivial: use g_malloc(), g_malloc0(), g_try_malloc(), g_try_malloc0(), and g_realloc().

Reply:
Good idea, but I am only going to define them for `GMalloc`, leaving them out of `GMallocStrv` because for now the intention is to only use the latter for the return value of `gKeyFileGetKeys()`.